handler/src/main/java/io/netty/handler/ssl/Java8SslUtils.java

```diff
@@ -29,6 +31,14 @@
 
     private Java8SslUtils() { }
 
+    static boolean isValidHostName(String hostname) {
```

Comment:
I think this can just be moved to SslUtils now as it has no java8 dependencies

Reply:
Done.